Example/Emission.xcodeproj/project.pbxproj

```diff
@@ -799,13 +804,15 @@
 				PRODUCT_BUNDLE_IDENTIFIER = net.artsy.Emission;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE_SPECIFIER = "match AppStore net.artsy.Emission";
+				SWIFT_VERSION = 4.2;
```

Comment:
This links swift libs in Emission but I think that's ok since we already are linking swift in Eigen